.github/workflows/ci.yml

```diff
@@ -7,39 +7,36 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        build: [pinned, stable, nightly]
```

Comment:
can you keep the pinned and figure out the earliest version that works? We use that to know if we require a new Rust version.

Reply:
Sure! Happy to do that 🙂